src/zoid/buttons/container.jsx

```diff
@@ -141,6 +143,8 @@ export function containerTemplate({ uid, props, tag, context, frame, prerenderFr
             <node el={ prerenderFrame } />
 
             <div id={ ID.SMART_MENU } class={ CLASS.SMART_MENU } />
+
+            <div id={ ID.INSTALLMENTS_MODAL } class={ CLASS.INSTALLMENTS_MODAL } />
```

Comment:
I think we can just render this modal up to `body` right? Without introducing a new container

Reply:
This actually helps in accessibility where keyboard tabbing moves immediately to installments popup after buttons. We can have a check before rendering if this container exists or else append to body